product_variant_cost_price/__init__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Ainara Galdona - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from . import models
+from openerp import SUPERUSER_ID
+
+
+def load_cost_price_on_variant(cr, registry):
+    product_obj = registry['product.product']
```

Comment:
Use this SQL expression for being multi-company aware:
```
        """
        INSERT INTO ir_property
            (name, type, value_float, value_integer, company_id, res_id,
             fields_id)
        SELECT
            ir_property.name,
            ir_property.type,
            ir_property.value_float,
            ir_property.value_integer,
            ir_property.company_id,
            'product.product,' || product_product.id::text,
            ir_model_fields.id
        FROM
            ir_property,
            product_product,
            ir_model_fields
        WHERE
            ir_property.name IN ('standard_price', 'cost_method')
        AND
            substring(ir_property.res_id from 18)::integer =
                product_product.product_tmpl_id
        AND
            ir_model_fields.model = 'product.product'
        AND
            ir_model_fields.name = ir_property.name
        ;
        """
```
The strange thing is that I haven't found any ir_property entry for cost_method. Can you investigate about it?